app/uk/gov/hmrc/apiplatformorganisationfrontend/models/CreateOrganisationRequest.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apiplatformorganisationfrontend.models
+
+import play.api.libs.json.{Format, Json, OFormat}
+
+final case class OrganisationName(value: String) extends AnyVal {
+  override def toString: String = value
+}
+
+object OrganisationName {
+  implicit val orgNameFormat: Format[OrganisationName] = Json.valueFormat[OrganisationName]
+}
+case class Organisation(id: String, organisationName: OrganisationName)
```

Comment:
Could `id` be a UUID?

Reply:
Discussed offline and agreed to postpone the decision until the backend design is more mature.